CMakeLists.txt

```diff
@@ -589,7 +601,7 @@ endif(QUDA_COVDEV)
 set(CMAKE_F_FLAGS -std=c99 CACHE STRING "Fortran Flags")
 
 # derive whether we need to build the fortran interface
-if(QUDA_INTERFACE_TIFR OR QUDA_INTERFACE_BQCD)
+if(QUDA_INTERFACE_TIFR OR QUDA_INTERFACE_BQCD OR QUDA_ARPACK)
```

Comment:
Why does `QUDA_ARPACK` being enabled mean we build QUDA's Fortran interface?

Reply:
Probably doesn't need it. I'll test it out without.